BrainPortal/app/views/layouts/_section_account.html.erb

```diff
@@ -56,7 +56,10 @@
         <% end %>
       <% end %>
     <% end %>
-    Rev:&nbsp;<%= $CBRAIN_StartTime_Revision %>
+    Rev: <%= CBRAIN::CBRAIN_StartTime_Revision %>
+    <% if current_user.has_role?(:admin_user) %>
```

Comment:
Why build a string concatenation in the substitution? Why not simply:

<% if current_user.has_role?(:admin_user) %>
    Branch: <%= CBRAIN::CBRAIN_Git_Branch.presence || "unknown" %>
<% end %>